Localization/ExplorerL10N.ini

```diff
@@ -1166,13 +1166,13 @@ Copy.No = &No
 Copy.YesAll = Sì t&utti
 Folder.Title = Conferma sostituzione cartella
 Folder.Prompt = Continuare?
-Toolbar.GoUp = Livello superiore
+Toolbar.GoUp = Cartella superiore
 Toolbar.Cut = Taglia
 Toolbar.Copy = Copia
 Toolbar.Paste = Incolla
 Toolbar.PasteShortcut = Incolla collegamento
 Toolbar.Delete = Elimina
-Toolbar.Email = Invia per posta elettronica gli elementi selezionati
+Toolbar.Email = Invia gli elementi selezionati via e-mail
```

Comment:
Google Translate:
Invia per posta elettronica gli elementi selezionati - Email the selected items
Invia gli elementi selezionati via e-mail - Send the selected items via e-mail

Reply:
For me should be

"Invia via email elementi selezionati"

No "e-mail" no article in the menu options

Reply:
I don't like how "Invia via email" sounds in italian.
I'd prefer "Invia gli elementi selezionati tramite e-mail"

Reply:
I prefer not use in the menu the article when not necessary (in English no one article) 
Then should be

"Invia elementi selezionati via email" (not e-mail)

Reply:
@Hadden89 

Is clear "via" as "tramite" and "e-mail" is not right.

Reply:
"Invia elementi selezionati via email" is fine. 
Only "invia/via" was not really nice to read :)
I proposed "tramite" just to avoid repeating "via" which is both in "inVIA" and in "VIA".
And yes, "gli", could be also omitted.